tests/fixture/e2e/flags.go

```diff
@@ -82,11 +87,17 @@ func RegisterFlags() *FlagVars {
 		false,
 		"[optional] reuse an existing network. If an existing network is not already running, create a new one and leave it running for subsequent usage.",
 	)
+	flag.BoolVar(
+		&vars.delayNetworkShutdown,
+		"delay-network-shutdown",
+		false,
+		"[optional] whether to delay network shutdown to allow a final metrics scrape. The duration is set with --network-shutdown-delay.",
+	)
 	flag.DurationVar(
 		&vars.networkShutdownDelay,
 		"network-shutdown-delay",
 		12*time.Second, // Make sure this value takes into account the scrape_interval defined in scripts/run_prometheus.sh
-		"[optional] the duration to wait before shutting down the test network at the end of the test run. A value greater than the scrape interval is suggested. 0 avoids waiting for shutdown.",
+		"[optional] the duration to wait before shutting down the test network at the end of the test run. A value greater than the scrape interval is suggested.",
 	)
```

Comment:
Why do we have 2 values for this? Seems like it would make more sense to just have the `--network-shutdown-delay` flag that defaults to `0`.

Reply:
If we're adding `--delay-network-shutdown` so that the user doesn't need to know the metrics scrape interval, is there ever a time we expect the user to specify the shutdown delay?

I'm wondering if we could remove `--network-shutdown-delay` if we don't want to specify `--network-shutdown-delay=12s` in the CI runs.